arrow-fx/src/test/kotlin/arrow/fx/MVarTest.kt

```diff
@@ -211,18 +213,19 @@ class MVarTest : UnitSpec() {
           }
         }
 
-        mvar.just(Option(0)).flatMap(::exec)
-          .unsafeRunSync() shouldBe count.toLong() * (count - 1) / 2
+        mvar.just(Option(0))
+          .flatMap(::exec)
+          .shouldBeEq(IO.just(count.toLong() * (count - 1) / 2), EQ())
       }
 
       "$label - producer-consumer parallel loop" {
-        fun producer(ch: Channel<Int>, list: List<Int>): IO<Unit> =
+        fun producer(ch: Channel<Long>, list: List<Long>): IO<Unit> =
```

Comment:
Why the change to `Long`?

Reply:
Because the EQ generated was trying to compare an Int and a Long and therefore failing

Reply:
I see.